src/Microsoft.AspNetCore.Http.Abstractions/Extensions/HeaderDictionaryExtensions.cs

```diff
@@ -11,6 +11,7 @@ public static class HeaderDictionaryExtensions
         /// <summary>
         /// Add new values. Each item remains a separate array entry.
         /// </summary>
+        /// <param name="headers">The <see cref="IHeaderDictionary"/> that this method extends.</param>
```

Comment:
Not sure I understand the current comment - this method isn't "extending the dictionary".

How about something like: `The <see cref="IHeaderDictionary"/> to append the specified values to.`?

@Tratcher can you comment on this comment?